client/components/domains/register-domain-step/index.jsx

```diff
@@ -525,6 +525,14 @@ const RegisterDomainStep = React.createClass( {
 				productSlug: suggestion.product_slug
 			} ) );
 
+			if ( abtest( 'privacyCheckbox' ) === 'checkbox' && (
```

Comment:
IMHO, this dangling bracket at the end of the line (`&& (`) and the additionally indented opening bracket for the `if` block at the end make for a bit confusing code :D
```
if ( abtest( 'privacyCheckbox' ) === 'checkbox' && (
	cartItems.isNextDomainFree( this.props.cart ) || cartItems.shouldBundleDomainWithPlan( domainsWithPlansOnlyTestEnabled, this.props.selectedSite, this.props.cart, suggestion )
	) ) {
	// some code
```
Maybe something like this could be better:
```
if ( abtest( 'privacyCheckbox' ) === 'checkbox' &&
	(
		cartItems.isNextDomainFree( this.props.cart ) ||
		cartItems.shouldBundleDomainWithPlan( domainsWithPlansOnlyTestEnabled, this.props.selectedSite, this.props.cart, suggestion )
	)
) {
	// some code
```
It's more lines, but seems more readable.
Or you could pull some of those props into `const`s and fit it into one line :wink:
```
const { cart, selectedSite } = this.props,
	willBundleDomainWithPlan = cartItems.shouldBundleDomainWithPlan( domainsWithPlansOnlyTestEnabled, selectedSite, cart, suggestion );
// (...)
if ( abtest( 'privacyCheckbox' ) === 'checkbox' && ( cartItems.isNextDomainFree( cart ) || willBundleDomainWithPlan ) ) {
	// some code